### ISSUE TITLE
[BUG](accessibility): spotify embed missing title attribute

### ISSUE THREAD

**Issue by ryandotfurrer** [TheTechCommute/TheTechCommute#31]
**Describe the bug**
spotify embed missing title attribute

**Expected behavior**
The `iframe` should have a `title` attribute describing the content of it.

**Screenshots**
![Arc Tech Commute-000112](https://github.com/user-attachments/assets/b3443699-3bdb-4f39-9bd4-fa77624ca04d)

**Comment by aasthajajoo123** [comment]
@ryandotfurrer hi! please assign this task to me.

**Comment by ryandotfurrer** [comment]
> @ryandotfurrer hi! please assign this task to me.
> 

I cannot, perhaps @jasonetorres or @elliezub can?

**Comment by aasthajajoo123** [comment]
@jasonetorres hi! can you please assign this task to me?

**Comment by swarupsahu08** [comment]
Hey @jasonetorres Can you assign me this issue
